python/private/common/py_executable_bazel.bzl

```diff
@@ -330,7 +330,7 @@ def _create_stage2_bootstrap(
         imports,
         runtime_details):
     output = ctx.actions.declare_file(
-        "{}_stage2_bootstrap.py".format(output_prefix),
+        "_{}_stage2_bootstrap.py".format(output_prefix),
```

Comment:
```suggestion
        # Prepend with underscore to prevent pytest from trying to
        # process the bootstrap for files starting with `test_`
        "_{}_stage2_bootstrap.py".format(output_prefix),
```